crates/bevy_render/src/camera/camera.rs

```diff
@@ -247,6 +247,8 @@ pub enum RenderTarget {
     Window(WindowId),
     /// Image to which the camera's view is rendered.
     Image(Handle<Image>),
+    /// Buffered Image to which the camera's view is rendered.
```

Comment:
I feel like this description could be clearer. The first image is used for rendering into and is then copied into the second image, right?